src/redux/books/actions.js

```diff
@@ -0,0 +1,34 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const APP_ID = 'rMCxybjWy2nJxNQ9XHZB';
+export const getBooks = createAsyncThunk('books/getBooks', async (URL, { rejectWithValue }) => {
+  try {
+    const data = await axios.get(`${URL + APP_ID}/books`);
+    return data.data;
+  } catch (error) {
+    return rejectWithValue(error.message);
+  }
+});
+
+export const addBook = createAsyncThunk('books/addBook', async ({ URL, newBook }, { rejectWithValue }) => {
+  try {
+    const data = await axios.post(`${URL + APP_ID}/books`, newBook, {
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+    return data.data;
+  } catch (error) {
+    return rejectWithValue(error.message);
+  }
+});
```

Comment:
- [ ] Thank you for addressing the eslint comment issue. However, the addBook functionality still needs some work as suggested by the previous reviewer as well. When I tried to add more than one book to the store, I got a forbidden error 403. Please check your code and make sure it allows multiple book additions without any errors.